changelogs/fragments/57837-add-preserve_time-to-get_url.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - get_url - Add preserve_time option to keep Last-Modifed datetime as mtime (https://github.com/ansible/ansible/issues/57837)
```

Comment:
```suggestion
  - get_url - Add preserve_time option to keep Last-Modified datetime as mtime (https://github.com/ansible/ansible/issues/57837).
```

Reply:
Hi @Akasurde,
Thank you for your review. I have fixed typos.